From 33950d0182fa788b66ef14079f24e9f5476b0948 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Thu, 10 Aug 2023 15:41:06 +0300
Subject: [PATCH] [SW-0] gaudi3: remove static from init_pcie

static directive is removed from pif_arc_init_pcie() so it can be called
by the SVE code and used for PCI flow tests by SIval team.

Change-Id: I00906ffbefa3c2da1b73350a7cd7e290df95060c
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 707561f..481e545 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -1051,7 +1051,8 @@
 
 #define LTTSM_ENA_MASK	(PCIE_AUX_LTSSM_EN_IND_M | PCIE_AUX_LTSSM_EN_MASK_M)
 
-static void pif_arc_init_pcie(void)
+/* not static because used by SVE */
+void pif_arc_init_pcie(void)
 {
 	pif_arc_init_pcie_phy();
 
diff --git a/arch/gaudi3/arc/hl.h b/arch/gaudi3/arc/hl.h
index 70e92ae..f3fbe40 100644
--- a/arch/gaudi3/arc/hl.h
+++ b/arch/gaudi3/arc/hl.h
@@ -50,5 +50,6 @@
 void pif_arc_pcie_monitoring(void);
 u32 gaudi3_die_timestamp_base_get(u32 die_id);
 int gaudi3_hbw_memcpy(u64 src, u64 dst, u32 size);
+void pif_arc_init_pcie(void);
 
 #endif /* ARCH_GAUDI3_ARC_HL_H_ */
